src/Forms/CompositeField.php

```diff
@@ -545,6 +545,7 @@ public function validate($validator)
             /** @var FormField $child */
             $valid = ($child && $child->validate($validator) && $valid);
         }
+        $this->extendValidationResult($valid, $validator);
         return $valid;
```

Comment:
```suggestion
        return $this->extendValidationResult($valid, $validator);
```
`$valid` isn't passed by reference to that method - plus we should do them all the same way

Reply:
Good spot!